### ISSUE TITLE
Videos playable by MPC but not listable by PMS

### ISSUE THREAD

**Issue by JackDandy** [rg3/youtube-dl#1652]
Hi,

Strange one...

Windows, git cloned youtube-dl (w/upgrade on first run)

In a command prompt, two files are downloaded. One from youtube and one from facebook. Defaults are used, that is ...
```
I:\youtube-dl\youtube-dl>python youtube-dl "http://www.facebook.com/photo.php?v=10151920533873077&set=vb.14936967018&type=2&theater"
[facebook] 10151920533873077: Downloading webpage
[download] Destination: BBC NEWS - Russell Brand on people, planet & revolution!-10151920533873077.mp4
[download] 100% of 38.94MiB in 00:2091MiB/s ETA 00:006
```
The new mp4 video files are copied into a folder where two older mp4 video files fetched using something else exist.

Now, here is the issue... when I "Refresh" Plex PMS, it will identify and list the older files but not the new ones fetched using yourtube-dl.  However, I ***can*** play the new files on a networked PC using MPC just fine.  Therefore, it seems like a tag or minor codec related anomoly is causing the files not to be identified by PMS. Files were renamed to short names in case that was an issue.

To be clear, I am not talking about being able to play the files, PMS wont even identify and **list** the files.

Can anybody help ?

```
I:\youtube-dl\youtube-dl>python youtube-dl -v
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['-v']
[debug] youtube-dl version 2013.10.23
[debug] Python version 2.7.2 - Windows-XP-5.1.2600-SP3
[debug] Proxy map: {}
Usage: youtube-dl [options] url [url...]
```

**Comment by yan12125** [comment]
Facebook videos are downloaded as-is. That is, the downloaded file is identical to that on the Facebook server. If PMS can't identify it, it's likely a problem in PMS. I'm closing this issue now as it's not a problem of youtube-dl. If you know how PMS works and youtube-dl needs a change for that, feel free to leave comments here or open a new issue.

**Comment by JackDandy** [comment]
Thanks. Better late than never.
It would have been better for someone to attempt the steps that I put together 3 years ago instead of this stock "can't be bothered" reply today, I gave detailed notes on tags added by YTDL, the file was modified post download - this was made clear in the detail I gave back then.  Anyway, I've long watched that video and that fb page still exists but I won't be spending anymore time on issues/feedback here as mine didn't get a reasonably timed response.  However, sincere thanks to whomever supports code on YTDL, I've used it a good few times over the years and still find it a very handy tool.
